packages/core/src/compiler/builtins/$richText/utils/getRichTextComponentConfigFragment.ts

```diff
@@ -11,34 +11,35 @@ import { stripRichTextPartSelection } from "./stripRichTextTextPartSelection";
 
 function getRichTextComponentConfigFragment(
   sourceRichTextComponentConfig: RichTextComponentConfig,
-  editorContext: EditorContextType
+  focussedField: EditorContextType["focussedField"],
+  locale: EditorContextType["contextParams"]["locale"],
+  formValues: EditorContextType["form"]["values"],
+  definitions: EditorContextType["definitions"]
```

Comment:
Wouldn't it be more correct to change dependency here from `EditorContextType` to `EasyblocksCanvasState`? I know they're the same for now but technically we operate on the canvas level here.